libs/libpqxx/Makefile

```diff
@@ -0,0 +1,53 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libpqxx
+PKG_VERSION:=7.3.0
+PKG_RELEASE:=1
+PKG_INSTALL:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/jtv/libpqxx
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MIRROR_HASH:=c176771dbbf02e38b84a8f9bf986a83587e2d86d9e0d2238e2a17a036e865951
+
+CMAKE_INSTALL:=1
+
+PKG_MAINTAINER:=Igor Bezzubchenko <garikello@gmail.com>
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/libpqxx
+	SECTION:=libs
+	CATEGORY:=Libraries
+	DEPENDS:=+libpq +libstdcpp
+	TITLE:=PostgreSQL client library (C++ interface)
+	URL:=http://pqxx.org/development/libpqxx
+	SUBMENU:=Database
+endef
+
+define Package/libpqxx/config
+	source "$(SOURCE)/Config.in"
+endef
+
+CMAKE_OPTIONS += \
+	-DBUILD_DOC=off \
```

Comment:
Please capitalize options. That's how it's done elsewhere.

Reply:
Fixed